docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -13,6 +13,15 @@
     - In the resulting window, press the 'Modify' button
     - Check the "Visual Studio Extensibility Tools Update 3" item and press the 'Next' button
     - Press the 'Update' button
+3. Install the [RC insider VSIX](https://dotnet.myget.org/F/roslyn/vsix/eb2680f2-4e63-44a8-adf6-2e667d9f689c-2.0.0.6110410.vsix).
```

Comment:
Is this `RoslynDeployment.vsix`, or `RoslynDeployment.Full.vsix`?  The difference is the diagnostics toolwindow package, which I don't think customers care about.